src/client/src/rt-components/adaptive-loader/AdaptiveLoader.tsx

```diff
@@ -30,10 +30,10 @@ interface BarProps {
   speed: number
   type: LoaderType
 }
-const Bar = styled('rect')<BarProps>`
+const Bar: StyledComponent<BarProps> = styled('rect')`
```

Comment:
`const Bar = styled.rect<BarProps>`
  animation: ${({ moveDistance }) => getBounce(moveDistance)} ${({ speed }) => speed}s infinite;
  animation-delay: ${({ order, speed }) => order * (speed / 1.3 / BAR_NUMBER)}s;
  fill: ${({ theme, type }) => theme.shell.textColor};
  will-change: transform;`

Reply:
No need for StyledComponent. If you do it this way it just gets inferred and everything works as you intended.

Reply:
No syntax completions or highlighting 😢 
<img width="764" alt="no-highlighting" src="https://user-images.githubusercontent.com/297455/44346158-491aa280-a463-11e8-9465-973a3b65bf1c.png">

Syntax completions and highlights! 🎉
<img width="793" alt="highlighting" src="https://user-images.githubusercontent.com/297455/44346160-4ae46600-a463-11e8-8796-09d2f66e275d.png">

Reply:
![screen shot 2018-08-20 at 15 31 05](https://user-images.githubusercontent.com/15194540/44346619-500ada80-a48e-11e8-848e-d840de6c0ec5.png)